File: PRATICA_1/circunferencia_retangulo.py
import math

opcao = input("Digite 'c' para calcular círculo ou 'r' para calcular retângulo: ")

if opcao == 'c':
    raio = float(input("Digite o raio do círculo: "))
    pi = 3.14
    circunferencia = 2 * pi * raio
    area = pi * raio ** 2
    print("O comprimento da circunferência é:", circunferencia)
    print("A área do círculo é:", area)

elif opcao == 'r':
    lado1 = float(input("Digite o valor do primeiro lado do retângulo: "))
    lado2 = float(input("Digite o valor do segundo lado do retângulo: "))
    perimetro = 2 * (lado1 + lado2)
    area = lado1 * lado2
    print("O perímetro do retângulo é:", perimetro)
    print("A área do retângulo é:", area)

else:
    print("Opção inválida!")

File: PRATICA_1/circunferencia_retangulo.c
/*

* Program: [Circunferencia e Retangulo]

* Author: [Matheus H Alves Medeiros]

* Date: [05/03/2024]

* Description: [Este progama realiza calculos reelacionados a circunferencia e retangulo a partir de dados atribuidos pelo usuário]

* License: [CC BY]

*/

#include <stdio.h>
#include <math.h>
#include <stdlib.h>
const float pi = 3.14;

typedef struct {
	
	float raio;
	float circunf;
	float area;
} circle;

int main (){
	
	int opcao;
	
	printf ("Digite 1 para calculos relacionados a circunferencia ou 2 para relacionados a retangulo\n");
	scanf ("%d", &opcao);
	
	if (opcao == 1){
		
		circle circulo1;
		
		printf ("Digite o valor do raio\n");
		scanf ("%f", &circulo1.raio);
		getchar ();
		
		circulo1.circunf = 2 * pi * circulo1.raio;
		circulo1.area = pi * (pow(circulo1.raio,2));
		
		printf ("O comprimento da circunferenica e: %.2f\n", circulo1.circunf);
		printf ("A medida da area e: %.2f\n", circulo1.area);
	}
	
	else if (opcao == 2) {
		
		float *base, *altura;
		float area, perimetro;
		
		base = (float*)malloc(sizeof(float));
		altura = (float*)malloc(sizeof(float));
		
		printf ("Digite o valor da base\n");
		scanf ("%f", &*base);
		getchar();
		
		printf ("Digite o valor da altura\n");
		scanf ("%f", &*altura);
		getchar();
		
		perimetro = (2* *base) + (2 * *altura);
		area = *base * *altura;
		
		printf ("O perimetro e: %.2f\n", perimetro);
		printf ("A area e: %.2f\n", area);
		
		return 0;
			
	}

}
